Profesja/Include/Profesja.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+#include <iostream>
+#include <vector>
+#include <string>
+
+struct Player
+{
+  unsigned int currentHp = 20;
+  unsigned int power = 1;
+  std::vector <std::string> handdeck = {"karta1", "karta2", "karta3"};
+  class Profession

```

Comment:
Projesja, Mag, Paladyn itd. ---> osobne klasy, a nie w środku Player. Oczywiście każda to osobny plik.